packages/cache/src/cache/cache.ts

```diff
@@ -13,6 +13,20 @@ function getCachePrefix(adapterIdentifiers: TAdapterIdentifiers) {
   return `@flopflip/${adapterIdentifiers}-adapter`;
 }
 
+export function encodeCacheContext(cacheContext: any) {
+  const encodedAsJson = JSON.stringify(cacheContext);
+
+  const hashCode = [...encodedAsJson].reduce(
```

Comment:
This is a suggestion at this point. Computing a hashCode might already be enough and is simple. SHA1 also would work but requires - to use native crypto - a site to run under HTTPs. MD5 is another option. Happy to hear feedback.